2_attributePoints.R

```diff
@@ -136,5 +132,12 @@ suppressWarnings(rm(tv,tvDataYear,tvDataYear.s, yrs, closestYear, vals, pa))
 
 # write it out ----
 points_attributed@proj4string <- projInfo
+# re-name table columns from filename to shrtNms$gridName
+shrtNms$fileName <- gsub(".tif$", "", shrtNms$fileName)
+for (n in 1:length(names(points_attributed))) {
+  if (names(points_attributed)[n] %in% shrtNms$fileName) {
+    names(points_attributed)[n] <- shrtNms$gridName[shrtNms$fileName == names(points_attributed)[n]][1]
+  }
+}
```

Comment:
why doing all this when you can use shortnames above?

Reply:
This is done because the `names(envStack)` carry over to the `points_attributed`, and the names with a "." in them are treated differently (starting line 102; these are the temporal variables).